From 881774bfa60789499ebdb1111345e6d9cba605fd Mon Sep 17 00:00:00 2001
From: Constantin Musca <constantin.musca@intel.com>
Date: Fri, 31 May 2013 21:02:32 +0300
Subject: Add TCP transport support to minadbd sideload mode

Add support for using "adb sideload" over TCP

For: AXIA-1950
Change-Id: I459e2b86d62b2d376264aa7cb1138ef643aded33
Signed-off-by: Constantin Musca <constantin.musca@intel.com>
---
 minadbd/adb.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/minadbd/adb.c b/minadbd/adb.c
index 948f4b7..7343685 100644
--- a/minadbd/adb.c
+++ b/minadbd/adb.c
@@ -386,12 +386,14 @@ int adb_main()
 
     init_transport_registration();
 
-    // The minimal version of adbd only uses USB.
     if (access("/dev/android_adb", F_OK) == 0) {
         // listen on USB
         usb_init();
     }
 
+    // listen on the default adb TCP port
+    local_init(DEFAULT_ADB_LOCAL_TRANSPORT_PORT);
+
     if (setgid(AID_SHELL) != 0) {
         fprintf(stderr, "failed to setgid to shell\n");
         exit(1);
-- 
1.7.4.1

